### DISCUSSION TITLE
Overriding Http method.

### DISCUSSION THREAD

**Discussion by mahmutbayri** [clue/framework-x#105] [Ideas]
Hi,

I have created a simple crud app here https://github.com/mahmutbayri/framework-x-database-crud . In order to handle put and delete method that are sent via HTML form i have created a simple middleware -> https://github.com/mahmutbayri/framework-x-database-crud/blob/master/src/Middleware/HttpMethodOverride.php. 

Symfony and Laravel frameworks use the same method -> https://github.com/symfony/http-foundation/blob/6.1/Request.php#L1200

Do you think that we should add this feature?

Thanks.

**Comment by SimonFrings** [comment]
@mahmutbayri looks great :+1: 

I've talked with @clue about this and we definitely see potential for this being a part of X one day. We had a similar approach already in mind so it's nice to see that we're on the same page here ^^

I want to use this opportunity to pass the same question back to you and the community. Do you think this should be shipped as part of X?
